spec/index.html

```diff
@@ -368,6 +368,39 @@ <h2>Introduction</h2>
     [[[SPARQL12-RESULTS-JSON]]] and 
     [[[SPARQL12-RESULTS-CSV-TSV]]].
   </p>
+
+      <section id="version-announcement">
+        <h3>Version Announcement</h3>
+        <p>
+        An optional version can be given to SPARQL results.
+        </p>
+        <p>
+        The `version` <a href="#mediatype">Media Type parameter</a> MAY be set to `1.2`
+        to indicate possible use of RDF 1.2 features in the SPARQL results.
+        Alternatively, the <a href="#docElement">`"version"` child element</a> MAY be used to specify this version.</p>
+        <p>Possible values for the version string are discussed in <a data-cite="RDF12-CONCEPTS#section-version-announcement"></a>.</p>
```

Comment:
FYI: The defined labels are at `#defined-version-labels` as discussed in https://github.com/w3c/rdf-concepts/issues/202#issuecomment-2842617173 and in PR https://github.com/w3c/rdf-concepts/pull/203.



The "Version Announcement" section is the "Introduction" section which is nonnormative.